docs/content/docs.md

```diff
@@ -630,13 +631,21 @@ although some may also have different parameters such as `DumpFlags`
 or just `string` and the help text explains that this will be
 interpreted as a list.
 
-### Time or duration options {#time-option}
+### Time options
 
-Options that take a `Duration` can be specified as a duration string
-or a time string.
+Options that take a `Time` parameter must be specified as an absolute
```

Comment:
I don't think is quite correct. `Time` options can take durations too, so `--default-time 1y` refers to a time 1 year ago.

`Duration` options can take times also so you can say `--min-age 2025-01-01`.

I don't think the original was quite correct either, but that is why it was only one section rather than two!

Reply:
Right. I did have a vague suspicion this could be a reason for the existing combined explanation. Will fix, i.e. change back to something more like it was.

Reply:
> I don't think the original was quite correct either

I found one error when testing these options a bit, but created a separate PR suggesting fixing functionality to make it match the documentation: https://github.com/rclone/rclone/pull/8318.